src/sentry/release_health/metrics.py

```diff
@@ -606,3 +601,342 @@ def extract_row_info(row: Mapping[str, Union[OrganizationId, str]]) -> ReleaseNa
+        session_status_column_name = tag_key(org_id, "session.status")
+
+        aggregates: List[SelectableExpression] = [
+            Column(release_column_name),
+            Column("project_id"),
+            Column(session_status_column_name),
+        ]
+
+        # Count of users and crashed users
+        rv_users: Mapping[Tuple[int, str, str], int] = {}
+
+        for row in raw_snql_query(
+            Query(
+                dataset=Dataset.Metrics.value,
+                match=Entity("metrics_sets"),
+                select=aggregates + [Column("value")],
+                where=where
+                + [
+                    Condition(Column("metric_id"), Op.EQ, metric_id(org_id, "user")),
+                    Condition(
+                        Column(session_status_column_name),
+                        Op.IN,
+                        get_tag_values_list(org_id, ["crashed", "init"]),
+                    ),
+                ],
+                groupby=aggregates,
+            ),
+            referrer="release_health.metrics.get_users_and_crashed_users_for_overview",
+        )["data"]:
+            key = (
+                row["project_id"],
+                reverse_tag_value(org_id, row[release_column_name]),
+                reverse_tag_value(org_id, row[session_status_column_name]),
+            )
+            rv_users[key] = row["value"]
+
+        return rv_users
+
+    @staticmethod
+    def _get_health_stats_for_overview(
+        where: List[Union[BooleanCondition, Condition]],
+        org_id: int,
+        health_stats_period: StatsPeriod,
+        stat: OverviewStat,
+        now: datetime,
+    ):
+        release_column_name = tag_key(org_id, "release")
+        session_status_column_name = tag_key(org_id, "session.status")
+        session_init_tag_value = tag_value(org_id, "init")
+
+        stats_rollup, stats_start, stats_buckets = get_rollup_starts_and_buckets(
+            health_stats_period
+        )
+
+        aggregates: List[SelectableExpression] = [
+            Column(release_column_name),
+            Column("project_id"),
+            Column("bucketed_time"),
+        ]
+
+        rv = defaultdict(lambda: _make_stats(stats_start, stats_rollup, stats_buckets))
+
+        for row in raw_snql_query(
+            Query(
+                dataset=Dataset.Metrics.value,
+                match=Entity(
+                    {
+                        "users": "metrics_sets",
+                        "sessions": "metrics_counters",
+                    }[stat]
+                ),
+                select=aggregates + [Column("value")],
+                where=where
+                + [
+                    Condition(
+                        Column("metric_id"),
+                        Op.EQ,
+                        metric_id(org_id, {"sessions": "session", "users": "user"}[stat]),
+                    ),
+                    Condition(Column("timestamp"), Op.GTE, stats_start),
```

Comment:
I would define a global variable in this file for all these column names since they are re-used quiet often to reduce typos

Reply:
sorry I really don't want to do that, I think this will make the code much harder to read. If there are typos then the query will immediately blow up, and ideally we'd eventually have enough type-safety to catch those errors otherwise.

Reply:
I feel like it will be hard to track where the error comes from, and its actually messier imo writing the name of the cols every single time 
I suggest doing something like this https://github.com/getsentry/sentry/blob/master/src/sentry/snuba/events.py for metrics

Reply:
I really don't agree. What's going on in this file is that we're mapping columns to discover search fields. There's no reference to `ERROR_MECHANISM` anywhere in the code.

What are you trying to solve? If somebody renames this column they'll have to have a extensive deprecation phase and probably can't rely on usage of this enum anyway unless enforced by the typechecker.

Reply:
I said something like not the exact same class, just so that 

1- It is easier to read 
2- Less typos
3- If for whatever reason, there is a schema change.. you don't need to go and change it in every place this column name is defined 

That being said, I leave the decision up to your team